code/mathematical-algorithms/armstrong_numbers/armstrong_number.php

```diff
@@ -0,0 +1,25 @@
+<?php
+function is_armstrong($number)
+{
+  $sum = 0;
+  $dupli = $number;
+  while($dupli != 0)
+  {
+    $digit = $dupli % 10;
+    $sum += $digit * $digit * $digit;
+    $dupli/=10;
+  }
+  if ($sum == $number)
+    return true;
```

Comment:
Simplify to `return $sum == $number;`

Reply:
@arnavb plz review my new commit [here](https://github.com/OpenGenus/cosmos/pull/2995)

Reply:
What about the rest of the changes?